can_tools/scrapers/official/AZ/az_demographics.py

```diff
@@ -0,0 +1,83 @@
+from tableauscraper import TableauScraper
+import us
+import pandas as pd
+from can_tools.scrapers import variables
+
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class ArizonaVaccineRace(StateDashboard):
+    has_location = False
+    source = "https://www.azdhs.gov/covid19/data/index.php#vaccine-admin"
+    source_name = "ARIZONA DEPARTMENT OF HEALTH SERVICES"
+    location_type = "county"
+    timezone = "US/Mountain"
+
+    state_fips = us.states.lookup("Arizona").fips
+
+    variables = {"total_vaccinations_initiated": variables.INITIATING_VACCINATIONS_ALL}
+    fetch_url = (
+        "https://tableau.azdhs.gov/views/VaccineDashboard/Vaccineadministrationdata"
+    )
+
+    def fetch(self):
+        engine = TableauScraper()
+        engine.loads(self.fetch_url)
+        return engine.getWorksheet(" County using admin county")
+
+    def normalize(self, worksheet: TableauScraper) -> pd.DataFrame:
+
+        # extract data from Tableau Dashboard
+        counties = [
+            t["values"]
+            for t in worksheet.getSelectableItems()
+            if t["column"] == "Admin Address County W/ State Pod"
+        ][0]
+        raw_data = []
+        for county in counties:
+            print("parsing", county)
+            workbook = worksheet.select("Admin Address County W/ State Pod", county)
+            raw_data.append(
+                workbook.getWorksheet("Race (# people)").data.assign(
+                    location_name=county
+                )
+            )
+        data = pd.concat(raw_data)
```

Comment:
nit: I assume concat isn't actually needed here?

Reply:
I think its needed as `raw_data` is a list of dataframes that then get combined into one here. We could probably make `raw_data` into a dataframe initially, and have the append in line 40 append to the dataframe directly (I'm not sure how that would compare performance wise). But, I think I'm going to leave it as-is for now if that's okay

Reply:
Ah, whoops! Yeah, I assumed wrong (I missed that raw_data was a list). My bad, thanks for explaining.  Carry on!